integration-tests/eks/validateResources_test.go

```diff
@@ -92,61 +104,64 @@ func TestK8s(t *testing.T) {
 	//Validating the Daemon Sets
 	daemonSets, err := ListDaemonSets(nameSpace, clientSet)
 	assert.NoError(t, err)
+	assert.Len(t, daemonSets.Items, 2)
 	for _, daemonSet := range daemonSets.Items {
 		fmt.Println("daemonSet name: " + daemonSet.Name + " namespace:" + daemonSet.Namespace)
+		// matches
+		// - cloudwatch-agent
+		// - fluent-bit
+		if match, _ := regexp.MatchString(agentName+"|fluent-bit", daemonSet.Name); !match {
```

Comment:
Can you run the EKS addon integ test after these changes

Reply:
https://github.com/aws/amazon-cloudwatch-agent-operator/actions/runs/6895006518

I had to create a new branch to run not-yet-merged GHA, but it's just a new branch off of this PR with  a change to run on a push to that temp branch.